sdk-setup/src/mb2

```diff
@@ -415,29 +416,31 @@ COMMANDS
         mandatory option:
 
         --pkcon
-            Uses 'pkcon' to install RPM package(s). Does not require root
-            access to the device. Not all device operating system versions and
-            configurations permit use of this method.
+            Uses 'pkcon' to install RPM packages.  Not all device operating
+            system versions and configurations permit use of this method.
 
         --rsync
-            This method is special.  Uses 'rsync' to copy the installed files
-            under '/opt/sdk/<name>' without building/installing RPM package(s).
-            Does not require root access to the device.  Incompatible with
-            '--all'.
+            This method is special.  It can be used to transfer the RPM
+            "buildroot" to the device as '/opt/sdk/<name>' without building
+            and installing RPM packages (see also 'make-install').
+            Incompatible with '--all'.
```

Comment:
I think the wording could still be improved. This can be interpreted as meaning "can be used without the need to build and install" while quite the opposite is true. It's not a big deal, as the command will now print a warning if buildroot is empty, so the user should realize his mistake at that point anyway.

Reply:
yeah that's because there are at least three context for "build and install" involved. I will think about improving it the next round - no idea at hand.

Reply:
handled in PR #500